newIDE/app/src/AssetStore/index.js

```diff
@@ -426,6 +427,28 @@ export const AssetStore = React.forwardRef<Props, AssetStoreInterface>(
       [isOnHomePage]
     );
 
+    const privateAssetPackFromSameCreator: ?Array<PrivateAssetPackListingData> = React.useMemo(
+      () => {
+        if (
+          !openedPrivateAssetPackListingData ||
+          !privateAssetPacks ||
+          !receivedAssetPacks
+        )
+          return null;
+
+        const receivedAssetPackIds = receivedAssetPacks.map(pack => pack.id);
+
+        return shuffle(
+          privateAssetPacks.filter(
+            pack =>
+              pack.sellerId === openedPrivateAssetPackListingData.sellerId &&
+              !receivedAssetPackIds.includes(pack.sellerId)
```

Comment:
Are we also displaying the pack they're currently viewing?
At first I was like "hmm useless", but actually it's a good way to show everything they've done to keep it.